content/SCALE/SCALETutorials/SystemSettings/Services/SMBServiceSCALE.md

```diff
@@ -27,18 +27,42 @@ Enter any alias name or names that do not exceed 15 characters in the **NetBIOS
 
 Enter a name that matches the Windows workgroup name in **Workgroup**. TrueNAS detects and sets the correct workgroup from these services when unconfigured with enabled Active Directory or LDAP active.
 
-If using SMB1 clients, select **Enable SMB1 support** to allow legacy SMB1 clients to connect to the server. Note: SMB1 is deprecated. We advise you to upgrade clients to operating system versions that support modern SMB protocol versions.
+If using SMB1 clients, select **Enable SMB1 support** to allow legacy SMB1 clients to connect to the server. Note: SMB1 is deprecated. We advise upgrading clients to operating system versions that support modern SMB protocol versions.
 
-If you plan to use the insecure and vulnerable NTLMv1 encryption, select **NTLMv1 Auth** to allow [smbd](https://www.samba.org/samba/docs/current/man-html/smbd.8.html) attempts to authenticate users. This setting enables backward compatibility with older versions of Windows, but we don't recommend it. Do not use on untrusted networks.
+If you plan to use the insecure and vulnerable NTLMv1 encryption, select **NTLMv1 Auth** to allow [smbd](https://www.samba.org/samba/docs/current/man-html/smbd.8.html) attempts to authenticate users.
+This setting enables backward compatibility with older versions of Windows, but we don't recommend it. Do not use on untrusted networks.
 
-Enter any notes about the service configuration in **Description**
+Enter any notes about the service configuration in **Description**.
 
 For more advanced settings, see [SMB Services Screen]({{< relref "SMBServicesScreen.md" >}}).
 
 Click **Save**.
 
 Start the **SMB** service.
 
+### Configuring Transport Encryption
+TrueNAS and Samba default behavior for SMB transport encryption allows SMB clients to negotiate different encryption levels for shares.
+This default setting enables negotiating encryption but does not turn on data encryption globally per share.
+SMB1 and SMB2 provide different settings to change the level of global or per-share SMB encryption applied to connections.
+See [Samba Server SMB Encrypt(s)](https://www.samba.org/samba/docs/current/man-html/smb.conf.5.html#SERVERSMBENCRYPT) for more information.
+
+You can change the SMB service to apply different SMB transport encryption levels to suit your use case. Go to the **SMB** service, found on the **System Settings > Services** screen, and click on **Advanced Settings**.
```

Comment:
```suggestion
You can change the SMB service to apply different SMB transport encryption levels to suit your use case.

Go to the **SMB** service, found on the **System Settings > Services** screen, and click on **Advanced Settings**.
```

Reply:
Also I think this is missing a step? 

Go to**System Settings > Services** screen > click the edit icon on the **SMB** service row > then click on **Advanced Settings**

Reply:
this needs to change to just System > Services so I'll edit the file.